prediction/sklearn/structured/custom_routines/prediction/predict.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# Copyright 2019 Google LLC. All Rights Reserved.
```

Comment:
`# Copyright 2019 Google LLC`
No periods or All Rights Reserved

Reply:
The period appears in all the files. Are you suggesting to remove all the periods from all the files?

Reply:
Yea.

Reply:
Done.